src/device/arm/interrupts.c

```diff
@@ -0,0 +1,22 @@
+#include <stdint.h>
+
+void EnableInterrupts(uintptr_t mask) {
+asm volatile(
+	"msr PRIMASK, %0"
+	:
+	: "r"(mask)
+	: "memory"
+);
```

Comment:
Can you indent this code? That looks better to me at least.
(...I've gotten so used to auto-formatted code that I feel weird about commenting about style issues).